recipes/ftfy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ftfy" %}
+{% set version = "4.4.2" %}
+{% set sha256sum = "0d82c3c7550d030c0b0e833f9b4d2a3c54873f7a934eec9913694977e3fb165f" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - ftfy = ftfy.cli:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    # Required for loading package data via `pkg_resources` module.
+    - setuptools
+    - html5lib
+    - wcwidth
+
+test:
+  imports:
+    - ftfy
+    - ftfy.build_data
+    - ftfy.chardata
+    - ftfy.cli
+    - ftfy.fixes
+    - ftfy.formatting
+
+  commands:
+    - ftfy -h
+
+about:
+  home: https://github.com/LuminosoInsight/python-ftfy/
+  license: MIT
+  summary: Fixes some problems with Unicode text after the fact
```

Comment:
This version does not include the `LICENSE.txt` file in the dist file. Development version does.

I've added the other url fields.